lib/foreman/renderer/scope/macros/base.rb

```diff
@@ -334,6 +334,21 @@ def gem_version_compare(first, second)
             Gem::Version.new(first.to_s) <=> Gem::Version.new(second.to_s)
           end
 
+          apipie :method, 'Returns a string that can be sorted' do
+            required :version, String, desc: 'Package version'
+            returns String, desc: 'String that can be sorted'
+            example 'sortable_version("1.5.2-1") #=> 01-1.01-5.01-2.01-1'
+            example 'sortable_version("") #=> ""'
+          end
+          # copied from katello Util::Package
+          def sortable_version(version)
```

Comment:
Could you please add some tests for this method showing that it works in various cases?

Reply:
Thanks, @tbrisker  for the help. I do not have a test environment configure. I updated the test case without running the test locally.
I hope the test will be green.

Reply:
@ranjan, could you please add at least one additional test (ideally 3) to test the comparison itself, since this macro is meant to do it?